src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -328,7 +333,7 @@ exports.createFiberFromText = function(content : string, priorityLevel : Priorit
   return fiber;
 };
 
-function createFiberFromElementType(type : mixed, key : null | string) : Fiber {
+function createFiberFromElementType(type : mixed, key : null | string, debugOwner : null | Fiber | ReactInstance) : Fiber {
```

Comment:
so the resulting fix is passing the `debugOwner` through the `createFiberFrom*` functions explicitly. Yeah?